config/honeybadger.php

```diff
@@ -165,4 +166,8 @@
          */
         'automatic' => HoneybadgerLaravel::DEFAULT_EVENTS,
     ],
+
+    'middleware' => [
+        AssignRequestId::class
```

Comment:
Should this be `Middleware\AssignRequestId::class` instead of `AssignRequestId::class`?

Reply:
No, because I imported the class (see line 6).